lib/generator.js

```diff
@@ -391,7 +393,7 @@ class Generator {
    */
   async configureTemplate() {
     if (isReactTemplate(this.templateConfig)) {
-      await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
+      await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION, this.compile);
```

Comment:
I'm wondering if it makes sense to modify `configureReact` or maybe better just do conditional here, like

```
if (isReactTemplate(this.templateConfig) && this.compile) {
      await configureReact(this.templateDir, this.templateContentDir, TRANSPILED_TEMPLATE_LOCATION);
```

Reply:
I have added conditional transpilation